src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/writefinaldecision/WriteFinalDecisionAboutToSubmitHandler.java

```diff
@@ -41,9 +46,62 @@ public boolean canHandle(CallbackType callbackType, Callback<SscsCaseData> callb
 
         PreSubmitCallbackResponse<SscsCaseData> preSubmitCallbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData);
 
-        //FIXME: check scores for mobility and daily living
+        // Obtain the list of points validation error messages (if any), and add to the response.
+        getDecisionNoticePointsValidationErrorMessages(sscsCaseData).forEach(preSubmitCallbackResponse::addError);
 
         return preSubmitCallbackResponse;
     }
 
+    private List<String> getDecisionNoticePointsValidationErrorMessages(SscsCaseData sscsCaseData) {
+
+        // For each PointsCondition, check if applicable for this submission.
+        // If so, extract any error messages generated
+        return Arrays.stream(PointsCondition.values())
+            .filter(pointsCondition -> pointsCondition.isApplicable(sscsCaseData))
+            .map(pointsCondition ->
+                getOptionalErrorMessage(pointsCondition, sscsCaseData))
+            .filter(Optional::isPresent)
+            .map(Optional::get)
+            .collect(Collectors.toList());
+
+    }
+
+    /**
+     * Obtain the points award for an activity question, given an SscsCaseData instance.
+     *
+     * @param sscsCaseData The SscsCaseData
+     * @param activityQuestionKey The key of an activity question.
+     * @return The points awarded for that question, given the SscsCaseData instance provided.
+     */
+    private int getPointsForActivityQuestionKey(SscsCaseData sscsCaseData, String activityQuestionKey) {
+        // Given an activity question key, obtain the answer to the activity question, and
```

Comment:
Not sure we need to repeat the comment here as you have similar in Java doc above

Reply:
Thanks Jack - as we've now discussed,  I've removed these comments in line with code style consistency